jhove-core/src/main/java/edu/harvard/hul/ois/jhove/handler/XmlHandler.java

```diff
@@ -4406,10 +4411,14 @@ private void writeAESTimeRange (String baseIndent,
 	}
     }
 
-    /* Clean up a URI string by escaping forbidden characters.
+    /**
+     * Clean up a URI string by escaping forbidden characters. 
      * We assume (perhaps dangerously) that a % is the start of
-     * an already escaped hexadecimal sequence. */
-    private String cleanURIString (String uri) {
+     * an already escaped hexadecimal sequence.
+     * @param uri
+     * @return
+     */
+    public String cleanURIString (String uri) {

```

Comment:
Shouldn't this method be static ?

Reply:
I'd not thought that clearly about it before @tledoux but I think you're right. Will pr a refactor.